plugins/basic/js/shotgun/constants.js

```diff
@@ -36,14 +36,14 @@ sg_constants.product_info = {
     // Photoshop
     PHSP: {
         display_name: "Photoshop",
-        tk_engine_name: "tk-photoshop",
+        tk_engine_name: "tk-photoshopcc",
```

Comment:
@thebeeland i am not sure i understand the workflow - i think you will end up having to build a plugin for each adobe engine for this to work - what the python process needs is not the name of the engine but ultimately the instance name from the environment configuration, so the current code basically just discards this value as it is passed down from the javascript. I think it should be removed to avoid confusion.